flink-dist/src/main/flink-bin/bin/flink-console.sh

```diff
@@ -58,7 +58,9 @@ esac
 
 FLINK_TM_CLASSPATH=`constructFlinkClassPath`
 
-log_setting=("-Dlog4j.configuration=file:${FLINK_CONF_DIR}/log4j-console.properties" "-Dlog4j.configurationFile=file:${FLINK_CONF_DIR}/log4j-console.properties" "-Dlogback.configurationFile=file:${FLINK_CONF_DIR}/logback-console.xml")
+log4j_config=`findLog4jConfiguration ${FLINK_CONF_DIR}`
```

Comment:
I guess the question is why do we still hard-code `${FLINK_CONF_DIR}` ? is there a reason why a user has to put the log files in the same folder where the original log4j.properties is? if the whole idea is to replace it with something else customized, shouldn't the goal be to support user-provided path & files?

Reply:
The knowledge of it being in Flink and "dedicated" to Flink could be useful in the future; e.g., we could think about it being modified at runtime via the WebUI to change the log level. You couldn't do this if the log file is at an arbitrary location (for security reasons).